ts/components/conversation/message/message-content/MessageReactBar.tsx

```diff
@@ -91,8 +94,7 @@ function useIsRenderedExpiresInItem(messageId: string) {
   return expiryDetails.expirationTimestamp;
 }
 
-function formatExpiry({ expirationTimestamp }: { expirationTimestamp: number }) {
-  const diffMs = expirationTimestamp - Date.now();
+function formatExpiry({ diffMs }: { diffMs: number }) {
```

Comment:
Not sure what diff means? Maybe rename it?

Reply:
renamed to formatTimeLeft

Reply:
and renamed the variable itself